homeassistant/components/daikin/sensor.py

```diff
@@ -87,6 +95,7 @@ class DaikinSensorEntityDescription(SensorEntityDescription, DaikinRequiredKeysM
         device_class=DEVICE_CLASS_ENERGY,
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         value_func=lambda device: round(device.last_hour_cool_energy_consumption, 2),
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
Energy cannot be a measurement, as it represents power usage over time (and thus not a measurement at a single point in time).